src/main/resources/application.yml

```diff
@@ -7,3 +7,10 @@ spring:
 clients:
   users-api-host: https://cgjresszgg.execute-api.eu-west-1.amazonaws.com/users
   teams-api-host: https://cgjresszgg.execute-api.eu-west-1.amazonaws.com/teams
+
+retry:
```

Comment:
We can improve this client call in order to be a fault tolerant service with spring-retry for 5XX http status response errors.